api4/team.go

```diff
@@ -1205,10 +1205,15 @@ func inviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 	if graceful {
 		cloudUserLimit := *c.App.Config().ExperimentalSettings.CloudUserLimit
 		var invitesOverLimit []*model.EmailInviteWithError
-		if c.App.Srv().License() != nil && *c.App.Srv().License().Features.Cloud && cloudUserLimit > 0 {
-			subscription, subErr := c.App.Cloud().GetSubscription()
+		if c.App.Srv().License() != nil && *c.App.Srv().License().Features.Cloud && cloudUserLimit > 0 && c.IsSystemAdmin() {
```

Comment:
Why are we checking if system admin here?

Reply:
It was coming from [a merge conflict](https://github.com/mattermost/mattermost-server/commit/ddd94397065497cbb9b3f126eb39224d010a0433#diff-f2bd3b9e6421d2541b06211437e7c68292b99ff7b23c6a3bb31b7f93eeb3bff0L1208) but it was the other way around, we have to remove it instead of having it there